Firestore/core/test/firebase/firestore/remote/serializer_test.cc

```diff
@@ -259,6 +270,23 @@ TEST_F(SerializerTest, EncodesString) {
   }
 }
 
+TEST_F(SerializerTest, EncodesTimestamps) {
+  std::vector<Timestamp> cases{
+      Timestamp(),  // epoch
```

Comment:
Optional nit: consider `{}` instead of `Timestamp()` for symmetry with the other cases.

Reply:
Done.